### DISCUSSION TITLE
Blackduck reports BDSA-2023-0791 (CWE-287) on etcd v3.5.6

### DISCUSSION THREAD

**Discussion by beards** [spf13/viper#1549] [Q&A] [answered]
Hi, 

As title, we get Blackduck report that etcd v3.5.6 contains a privilege escalation vulnerability. 
May I know that do you have plan to upgrade it?
Blackduck said that v3.5.8 should be OK.

**Comment by sagikazarmark** [comment, marked as answer]
Thanks for reporting! We will update etcd soon.

**Comment by paulohenriquesg** [comment]
Hi all!

Do you have some timeframe when this update is going to happen? Is there anything we can do to help?

Thank you! :)

**Reply by sagikazarmark** [reply]
@paulohenriquesg thanks for the ping!

I updated the library on the main branch. I'm going to tag a new release in the coming days.
